paddle/fluid/operators/distributed/send_recv.proto.in

```diff
@@ -24,6 +24,7 @@ service SendRecvService {
   rpc SendVariable(VariableMessage) returns (VoidMessage) {}
   // Argument VariableMessage for GetVariable should only contain varname.
   rpc GetVariable(VariableMessage) returns (VariableMessage) {}
+  rpc GetVariableNoBarrier(VariableMessage) returns (VariableMessage) {}
```

Comment:
Adding a new RPC call may need to add a lot of code, and a new server-side thread pool. maybe can add one field in `VariableMessage` to indicate the request type.

Reply:
It will hang at sometimes.